### ISSUE TITLE
Why does it create an <none> image?

### ISSUE THREAD

**Issue by ghost** [sbt/sbt-native-packager#1229]
## steps

1. `sbt docker:publishLocal` 

## problem

There's a 703MB image named `<none>` that is created.

## expectations

sbt-native-packager cleans after itself, so `<none>` is deleted if it's only used for prep work.

## original bug report

Hi all 

When I run the command: 

`sbt docker:publishLocal`  

it always create the following images: 

```
sapmock             0.1                 db261ab225b5        8 hours ago         664MB
<none>              <none>              0940c75d8e5a        8 hours ago         703MB
openjdk             8                   bec43387959a        10 days ago         625MB
```
and I could not figure out, why it always create an `<none>` image. 

I am using the latest version of sbt-native-packager and the build.sbt looks as the following:

```
enablePlugins(JavaServerAppPackaging)
enablePlugins(DockerPlugin)
dockerExposedPorts := Seq(8080)

```
OS is UBUNTU 18.04  

What am I doing wrong? 

Thanks

**Comment by eed3si9n** [comment]
See https://github.com/sbt/sbt-native-packager/releases/tag/v1.3.16.
I added multi-stage Docker build in https://github.com/sbt/sbt-native-packager/pull/1190 (to change the file permission without fs layer overhead), so maybe it's that?

**Comment by vladimirshin** [comment]
+1, is there any progress on this issue?

**Comment by muuki88** [comment]
Thanks for asking @vladimirshin :smiley: . I'm happy to give you guidance and merge pull requests if you would like to work on this issue :hugs:

**Comment by yingding** [comment]
+1, do i need this <none> image? is it safe to be deleted or is there any dependencies on that <none> image?

**Comment by eed3si9n** [comment]
The image is used basically as a preparation kitchen to change your real image's file permissions. Without using the technique, it will double the final image size for no reason. If `<none>` is confusing, maybe we should try to name it or something, but I think it's a good technique, unless there are some other better technique to change permissions.

As far as I know, you should be able to delete it afterwards. Maybe that's something we can automate too.

**Comment by eed3si9n** [comment]
I rewrote the description of the bug. I hope @bifunctor won't mind.

**Comment by muuki88** [comment]
@mkurz thanks for updating this. Would you like to try to make that change? :smiley:

**Comment by mkurz** [comment]
Yes I will. PR is coming soon.

**Comment by mkurz** [comment]
Pull request with the fix: #1279
